commons-text/pom.xml

```diff
@@ -152,8 +152,8 @@
 										<version>${core.artifact.version}</version>
 									</artifact>
 									<moduleInfoSource>
-										open module org.apache.commons.text {
-											requires org.apache.commons.lang3;
+										open module dev.ikm.jpms.commons.text {
+											requires dev.ikm.jpms.commons.lang3;
```

Comment:
Same as before, the name of the module has been changed to `dev.ikm.jpms.commons.text`. Is this just an arbitrary convention for Apache Commons libs following the auto naming convention which is `org.apache.commons.text`? Is there any specific reason?

Reply:
see above - same concept